addons/addon-base-raas/packages/base-raas-services/lib/environment/service-catalog/environment-sc-connection-service.js

```diff
@@ -178,6 +178,15 @@ class EnvironmentScConnectionService extends Service {
   }
 
   async getRStudioUrl(requestContext, id, connection) {
+    // The username for RStudio url access was changed as of 08/26/20
+    // If you're experiencing difficulty accessing previously provisioned RStudio instance,
+    // the steps to perform for backwards compatibility are:
+    // 1. Redeploy machine-images SDC (this will update the RStudio AMIs to have the new username)
+    // 2. For already provisioned RStudio instances, get sudo/root access on the box and add the user
+    // as a linux user using the command 'sudo useradd -m rstudio-user'
+    // 3. Update the username in boot script 'set-password' (found in /usr/local/bin/)
+    // 4. Reboot the box
+
```

Comment:
Should we add this documentation in machine-images README where we have documentation on RStudio setup?

Reply:
Agreed. Done.